definitions/json/CaseEvent.json

```diff
@@ -470,6 +470,17 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "initialConsideration",
+    "Name": "Initial Consideration",
+    "Description": "Start page of the Initial Consideration process",
```

Comment:
Very minor: Might want to change the description as this event will be used for the entire initial consideration flow and not just start page. Maybe just to Initial Consideration

Reply:
Thanks, now updated.